gensim/test/test_phrases.py

```diff
@@ -184,6 +188,15 @@ def setUp(self):
         bigram_unicode_phrases = Phrases(unicode_sentences, min_count=1, threshold=1)
         self.bigram_unicode = Phraser(bigram_unicode_phrases)
 
+    def testRecodeToUtf8False(self):
+        """Test that Phraser works as expected when `recode_to_utf8 = False` """
+        expected = ['survey', 'user', 'computer', 'system', 'response_time']
+
+        bigram_recode_false = Phrases(sentences, recode_to_utf8=False, min_count=1, threshold=1)
+
+        bigram_phraser = Phraser(bigram_recode_false)
+        self.assertEqual(bigram_phraser[sentences[1]], expected)
+
```

Comment:
Please add tests for input in bytes and utf-8 and errors for a mixture of the two